lib/apr/views/commerce_transaction_created_slack_view.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Apr.Views.CommerceTransactionCreatedSlackView do
+  import Apr.Views.Helper
+
+  def render(subscription, event, routing_key) do
+    case {subscription.theme, event["properties"]["external_type"], event["verb"], event["properties"]["transaction_type"], event["properties"]["order"]["payment_method"], routing_key} do
+      {"dispute", "payment_intent", "created", "dispute", "us_bank_account", "dispute"} ->
```

Comment:
Is the `routing_key` here `"dispute"` or `"transaction.created"`? And actually, the `routing_key` was already matched in `Apr.Views.CommerceSlackView` above, so maybe it's not necessary to match it again.

Reply:
My bad 😞 The `routing_key` should be `"transaction.created` and since it's already matched in `Apr.Views.CommerceSlackView`, it's not necessary to match it again. Updated it here: https://github.com/artsy/aprd/pull/352/commits/cd2606d8b438827798d12c425da482fbac34fe05